### PR TITLE
Enhancement/kirby list accesibility

### PR DESCRIPTION
**Pull request by PetarRing**

### CODE REVIEW

File: src/kirby/components/list/list.component.scss [hunk 1]
```diff
@@ -38,4 +36,10 @@ ul {
       }
     }
   }
+}
+
+.clickable {
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.scss]: Can be moved up to line 20 under the `li` rule and add a `&`.
```suggestion
&.clickable {
...
}
```

I expect the class `clickable` is going to be used on anything else the `li` tag?

File: src/kirby/components/list/list.component.scss [hunk 2]
```diff
@@ -35,11 +35,14 @@ ul {
         background-color: transparent;
       }
     }
+
+    &.clickable {
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.scss]: Maybe the class should also be renamed. Sorry missed that in the last review.

File: tsconfig.json [hunk 3]
```diff
@@ -20,6 +20,10 @@
         "paths": {
             "~/*": [
                 "src/*"
+            ],
```
Comment (jonasjensen-brolstark) [inline tsconfig.json]: And you are sure on what you are doing here?

File: src/kirby/components/list/list.component.scss (hunk 2 above)
Comment (PetarRing) [reply src/kirby/components/list/list.component.scss]: We can do that. But this class is purely for web. If you think I will :)

Comment (PetarRing) [reply src/kirby/components/list/list.component.scss]: *will rename it

File: src/kirby/components/list/list.component.html [hunk 4]
```diff
@@ -11,7 +11,7 @@
         >
         </ng-container>
       </li>
-      <li class="row" *ngFor="let item of section.items" (click)="onItemClick(item)" >
+      <li class="row" [ngClass]="{'clickable': isSelectable}" *ngFor="let item of section.items" (click)="onItemClick(item)" tabindex="0">
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.html]: Maybe we should use the [Aria button role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) on the `li` tags that are selectable.

File: src/kirby/components/list/list.component.scss (hunk 2 above)
Comment (jonasjensen-brolstark) [reply src/kirby/components/list/list.component.scss]: Do it 🚀 

But it is just to use the same  vocabulary in code and styling.

File: src/kirby/components/list/list.component.html (hunk 4 above)
Comment (PetarRing) [reply src/kirby/components/list/list.component.html]: I belive I changed that and pushed it with a few other changes

**Pull request merged**